apps/docs/src/pages/theming/create-styles.md

```diff
@@ -56,15 +57,17 @@ The `createStyles` function theme object differs from the general theme object.
 
 ## Style Parameters
 
-You can receive any amount of parameters as second argument of createStyles function, later you will need to pass those parameters as argument to useStyles function:
+You can receive any amount of parameters as second argument of `createStyles` function, later you will need to pass those parameters as argument to `useStyles` function:
 
 <Demo demo={CreateStylesDemos.params} />
 
 ## Merging Classes (cx function)
 
 To merge class names use the `cx` function, it has the same API as classnames package.
 
-!important: Do not use external libraries like classnames or clsx with class names created with createStyles function as it will produce styles collisions.
+<Alert icon={InfoCircled}  title="Tip">
+    Do not use external libraries like `classnames` or `clsx` with class names created with `createStyles` function as it will produce styles collisions.
```

Comment:
```suggestion
    Do not use external libraries like `classnames` or `clsx` with class names created with `createStyles` function as it will produce style collisions.
```